app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-header.scss

```diff
@@ -19,10 +19,12 @@
   @include govuk-font(19, $weight: bold);
 }
 
+// stylelint-disable declaration-no-important
 @include govuk-media-query($media-type: print) {
   .gem-c-step-nav-header {
     padding: 0 0 govuk-spacing(4) 0;
-    background: none;
+    background: unset;
+    border-color: $govuk-print-text-colour !important;
```

Comment:
Not sure the `important` is necessary here?

Reply:
@andysellick The `!important` is now removed.